### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 99) …

### PR DESCRIPTION
**Pull request by alvarocamillont**
Fixes https://github.com/GoogleChrome/developer.chrome.com/issues/2151

Changes proposed in this pull request:

Add portuguese translations for What's New in DevTools M99

### CODE REVIEW

File: site/_includes/partials/devtools/pt/whats-new.md [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
-* [Display worker source files in the Sources panel](/pt/blog/new-in-devtools-99/#worker-sourcemap)
-* [Chrome’s Auto Dark Theme updates](/pt/blog/new-in-devtools-99/#auto-dark-mode)
-* [Touch-friendly color-picker and split pane](/pt/blog/new-in-devtools-99/#touch-friendly)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-99/#misc) -->
+### Chrome 99 {: #chrome99 }
+
+* [Limitando requests de WebSocket](/pt/blog/new-in-devtools-99/#websocket)
+* [Novo painel API de relatórios no painel Aplicativo](/pt/blog/new-in-devtools-99/#reporting-api)
+* [Suporte para esperar até que o elemento esteja visível/clicável no painel Gravação](/pt/blog/new-in-devtools-99/#recorder)
+* [Melhor estilização do console, formatação e filtragem](/pt/blog/new-in-devtools-99/#console)
```
Comment (khaosdoctor) [inline site/_includes/partials/devtools/pt/whats-new.md:11]:
```suggestion
* [Melhor estilização, formatação e filtragem no console](/pt/blog/new-in-devtools-99/#console)
```

File: site/_includes/partials/devtools/pt/whats-new.md [hunk 2]
```diff
@@ -3,18 +3,18 @@
...
+* [Depurar extensão do Chrome com arquivos de sourcemap](/pt/blog/new-in-devtools-99/#extension)
```
Comment (khaosdoctor) [inline site/_includes/partials/devtools/pt/whats-new.md:12]:
```suggestion
* [Debug de extensões do Chrome com arquivos de sourcemap](/pt/blog/new-in-devtools-99/#extension)
```

File: site/_includes/partials/devtools/pt/whats-new.md [hunk 3]
```diff
@@ -3,18 +3,18 @@
...
+* [Árvore de pastas aprimorada no painel Fontes](/pt/blog/new-in-devtools-99/#source-tree)
+* [Exibição arquivos fontes de workers no painel Origens](/pt/blog/new-in-devtools-99/#worker-sourcemap)
+* [Atualizações do Auto Dark Theme do Chrome](/pt/blog/new-in-devtools-99/#auto-dark-mode)
```
Comment (khaosdoctor) [inline site/_includes/partials/devtools/pt/whats-new.md:15]:
```suggestion
* [Atualizações na funcionalidade de tema escuro automático do Chrome](/pt/blog/new-in-devtools-99/#auto-dark-mode)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 4]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/Qf8f2x1u1y5FEMSmkB3A.mp4", class="screenshot", autoplay=true, controls=true, loop=true, muted=true %}
 
-Chromium issue: [1257499](https://crbug.com/1257499#c38)
+Issue Relacionada: [1257499](https://crbug.com/1257499#c38)
 
 
 <!-- ## Better console styling, formatting and filtering {: #console } -->
-
+## Melhor estilização do console, formatação e filtragem {: #console }
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-99/index.md:76]:
```suggestion
## Melhor estilização, formatação e filtragem do console {: #console } 
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 5]
```diff
@@ -29,164 +28,183 @@ draft: true
...
-Chromium issue: [1068788](https://crbug.com/1068788)
+Issue Relacionada: [1068788](https://crbug.com/1068788)
 
 
 <!-- ## Sourcemaps improvements {: #sourcemap } -->
-
+ ## Melhorias nos sourcemaps {: #sourcemap }
 <!-- ### Debug Chrome extension with sourcemap files {: #extension } -->
-
+### Depurar extensão do Chrome com arquivos de sourcemap  {: #extension }
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-99/index.md:125]:
```suggestion
### Depuração de extensões do Chrome com arquivos de sourcemap  {: #extension }
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 6]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/6e877d5e1a3ccca22e866fb2a70330613aeb6964 #}
 
-Chromium issue: [1277002](https://crbug.com/1277002) 
+Issue Relacionada: [1277002](https://crbug.com/1277002) 
 
 
 <!-- # Chrome’s Auto Dark Theme updates {: #auto-dark-mode } -->
-
+# Atualizações do Auto Dark Theme do Chrome {: #auto-dark-mode }
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-99/index.md:161]:
```suggestion
# Atualizações do Tema Escuro Automático do Chrome {: #auto-dark-mode }
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 7]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- The [Auto Dark Theme emulation](/blog/new-in-devtools-96/#auto-dark-mode) UI is now simplified. It is a checkbox now, it was a dropdown previously. -->
+A interface do [Auto Dark Theme](/blog/new-in-devtools-96/#auto-dark-mode) agora está simplificada. Agora é uma caixa de seleção,  anteriormente era uma lista suspensa.
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-99/index.md:163]:
```suggestion
A interface do [Tema Escuro Automático (Auto Dark Theme)](/blog/new-in-devtools-96/#auto-dark-mode) agora está simplificada. Agora é uma caixa de seleção, anteriormente era uma lista suspensa.
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 8]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 
 <!-- Apart from that, when the [Auto Dark Theme](/blog/auto-dark-theme/) is enabled, the **Emulate perfers-color-scheme** dropdown will be disabled and set to **perfers-color-scheme: dark** automatically. -->
+Além disso, quando o [Auto Dark Theme](/blog/auto-dark-theme/) estiver ativado, o menu suspenso **Emular perfers-color-scheme** será desativado e definido como **perfers-color-scheme : escuro** automaticamente.
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-99/index.md:166]:
```suggestion
Além disso, quando o [Auto Dark Theme](/blog/auto-dark-theme/) estiver ativado, o menu suspenso **Emular prefers-color-scheme** será desativado e definido como **prefers-color-scheme: escuro** automaticamente.
```

**Review by khaosdoctor — COMMENTED** [review]
Small reviews

**Comment by alvarocamillont** [comment]
Done!

**Comment by chybie** [comment]
Thank you both! @andreban PTAL.

File: site/pt/blog/new-in-devtools-99/index.md [hunk 9]
```diff
@@ -29,164 +28,183 @@ draft: true
...
-
+O painel **Rede** agora é compatível com a limitação de requests de web sockets. Anteriormente, a limitação de rede não funcionava em requests de web sockets.
 <!-- Open the **Network** panel, click on a web socket request and open the **Messages** tab to observe the message transfers. Select **Slow 3G** to throttle the speed.  -->
+Abra o painel **Rede**, clique em uma request de web socket e abra a guia **Mensagens** para observar as transferências de mensagens. Selecione **3G Lento** para limitar a velocidade.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ZHJibovD0IRQ7KrWb0aD.png", alt="Throttling WebSocket requests", width="800", height="540" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ZHJibovD0IRQ7KrWb0aD.png", alt="Limitando requests de WebSocket", width="800", height="540" %}
 
-Chromium issue: [423246](https://crbug.com/423246)
+Issue Relacionada: [423246](https://crbug.com/423246)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:39]:
```suggestion
Issue relacionada: [423246](https://crbug.com/423246)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 10]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- The **Endpoints** section gives you an overview of all the endpoints configured in the `Reporting-Endpoints` header.  -->
+A seção **Endpoints** fornece uma visão geral de todos os endpoints configurados no cabeçalho `Reporting-Endpoints`.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/D1fUz4zuS1xwDbszgft1.png", alt="Reporting API pane", width="800", height="560" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/D1fUz4zuS1xwDbszgft1.png", alt="Painel da API de relatórios", width="800", height="560" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/668bc7a4bc6bea854e8fc21f0e0ca3953ff5e95a #}
 
-Chromium issue: [1205856](https://crbug.com/1205856)
+Issue Relacionada: [1205856](https://crbug.com/1205856)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:60]:
```suggestion
Issue relacionada: [1205856](https://crbug.com/1205856)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 11]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- Here is an example of an off-screen menu positioned outside of the viewport and slide in when activated. The user flow is to toggle the menu, and click on the menu item. Previously, the replay would fail at the last step, because the menu item is still sliding in and not visible in the viewport yet. It’s fixed now. -->
+Aqui está um exemplo de um menu off-screen posicionado fora da janela de visualização e deslizar quando ativado. O fluxo do usuário é alternar o menu e clicar no item de menu. Anteriormente, a repetição falhava na última etapa, porque o item de menu ainda está deslizando e ainda não está visível na janela de visualização. Está corrigido a partir dessa versão.
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/Qf8f2x1u1y5FEMSmkB3A.mp4", class="screenshot", autoplay=true, controls=true, loop=true, muted=true %}
 
-Chromium issue: [1257499](https://crbug.com/1257499#c38)
+Issue Relacionada: [1257499](https://crbug.com/1257499#c38)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:72]:
```suggestion
Issue relacionada: [1257499](https://crbug.com/1257499#c38)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 12]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- In addition, if a console message is shown, the group (or the ancestor group) it belongs to is now shown as well.  -->
+Além disso, se uma mensagem do console for exibida, o grupo (ou o grupo ancestral) ao qual ela pertence agora também é exibido.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7iE7r79DI3cQxObhiZUh.png", alt="console group filter", width="800", height="612" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7iE7r79DI3cQxObhiZUh.png", alt="filtro de grupo de console", width="800", height="612" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/94734947c48283a56f93430f23b838cef10fd364 #}
 
-Chromium issue: [1068788](https://crbug.com/1068788)
+Issue Relacionada: [1068788](https://crbug.com/1068788)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:119]:
```suggestion
Issue relacionada: [1068788](https://crbug.com/1068788)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 13]
```diff
@@ -29,164 +28,183 @@ draft: true
...
+### Depuração de extensões do Chrome com arquivos de sourcemap  {: #extension }
 <!-- You can now [debug Chrome extension](/docs/extensions/mv3/getstarted/#unpacked) with sourcemap files. Previously, DevTools only supported inline sourcemap for Chrome extension debugging. -->
+Agora você pode [depurar a extensão do Chrome](/docs/extensions/mv3/getstarted/#unpacked) com arquivos de sourcemap. Anteriormente, o DevTools suportava apenas o sourcemap inline para depuração de extensão do Chrome.
 
 {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/lnRa954ROl0MSSExlBl7.png", alt="Debug Chrome extension with sourcemap files", width="800", height="518" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/1e73eb62955de7c4b0920575c7b374d47dab6a65 #}
 
-Chromium issue: [212374](https://crbug.com/212374)
+Issue Relacionada: [212374](https://crbug.com/212374)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:133]:
```suggestion
Issue relacionada: [212374](https://crbug.com/212374)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 14]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- The source folder tree in the **Sources** panel is now improved with less clutter in the folder structures and naming (e.g. “../”, “./”, etc). Under the hood, this is the result of normalizing the absolute source URLs in the sourcemaps. -->
+A árvore de pastas no painel **Fontes** agora foi aprimorada com menos confusão nas estruturas de pastas e nomenclatura (por exemplo, “../”, “./”, etc). Sob o capô, isso é o resultado da normalização das URLs de origem absolutas nos sourcemaps.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Wl1pPVfQ51NaCtpp3KuY.png", alt="Improved source folder tree in the Sources panel", width="800", height="444" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Wl1pPVfQ51NaCtpp3KuY.png", alt="Árvore de pastas aprimorada no painel Fontes", width="800", height="444" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/68613ab02f6d556a2c5ac68ea08f466a534c6bd9 #}
 
-Chromium issue: [1284737](https://crbug.com/1284737)
+Issue Relacionada: [1284737](https://crbug.com/1284737)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:145]:
```suggestion
Issue relacionada: [1284737](https://crbug.com/1284737)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 15]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- [Worker](https://web.dev/workers-overview/) (e.g. web worker, service worker) source files with relative SourceURL are now displayed in the **Source** panel. Previously, worker source files were not handled correctly. -->
+[Worker](https://web.dev/workers-overview/) (por exemplo, web worker, service worker) arquivos fontes com SourceURL relativo agora são exibidos no painel **Fontes**. Anteriormente, os arquivos fontes do worker não eram tratados corretamente.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/apH5n92bqYWINMQn5VXa.png", alt="ALT_TEXT_HERE", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/apH5n92bqYWINMQn5VXa.png", alt="Exibição arquivos fontes de workers no painel Origens", width="800", height="509" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/6e877d5e1a3ccca22e866fb2a70330613aeb6964 #}
 
-Chromium issue: [1277002](https://crbug.com/1277002) 
+Issue Relacionada: [1277002](https://crbug.com/1277002)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:157]:
```suggestion
Issue relacionada: [1277002](https://crbug.com/1277002) 
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 16]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- Chrome 96 introduces an [Origin Trial](/blog/origin-trials/) for [Auto Dark Theme](/blog/auto-dark-theme/) on Android. With this feature, the browser applies an automatically generated dark theme to light themed sites, when the user has opted into dark themes in the Operating System. -->
+O Chrome 96 apresentava uma [Origin Trial](/blog/origin-trials/) para [Auto Dark Theme](/blog/auto-dark-theme/) no Android. Com esse recurso, o navegador aplica um tema escuro gerado automaticamente a sites com temas claros, quando o usuário optou por temas escuros no sistema operacional.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/eqfY1jZI8kY7BknnuAom.png", alt="Auto Dark Theme emulation", width="800", height="476" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/eqfY1jZI8kY7BknnuAom.png", alt="Emulação Auto Dark Theme", width="800", height="476" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/8443d2894b6401695ce94657e6afd5ad399eef28 #}
 
-Chromium issue: [1243309](https://crbug.com/1243309)
+Issue Relacionada: [1243309](https://crbug.com/1243309)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:175]:
```suggestion
Issue relacionada: [1243309](https://crbug.com/1243309)
```

File: site/pt/blog/new-in-devtools-99/index.md [hunk 17]
```diff
@@ -29,164 +28,183 @@ draft: true
...
 <!-- Here is an example captured with the [Google Pixelbook](https://www.google.com/chromebook/device/google-pixelbook/) device touchscreen. -->
-
+Aqui está um exemplo capturado com a tela sensível ao toque do dispositivo [Google Pixelbook](https://www.google.com/chromebook/device/google-pixelbook/).
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/aA3Oann2z26Yty9sgNB2.mp4", class="screenshot", autoplay=true, controls=true, loop=true, muted=true %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/f60936b29519e0cf387cd0a133d43885c6eb183d #}
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/22bb84d657aa69f6f7d5067605c2c133a5714172 #}
 
-Chromium issues: [1284245](https://crbug.com/1284245), [1284995](https://crbug.com/1284995) 
+Issue Relacionadas: [1284245](https://crbug.com/1284245), [1284995](https://crbug.com/1284995)
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-99/index.md:189]:
```suggestion
Issues relacionadas: [1284245](https://crbug.com/1284245), [1284995](https://crbug.com/1284995) 
```

**Review by andreban — APPROVED** [review]
A few nits only

**Pull request merged**